### PR TITLE
add confirm email page

### PR DESCRIPTION
**Pull request by Derick1530**

### CODE REVIEW

File: components/Auth/ConfirmEmail/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+const ConfirmEmail = () => {
+    return (
+        <div className="auth h-screen">
+            <div className='flex justify-center items-center w-full h-full'>
+                <div className='lg:max-w-[40%] px-5 w-full bg-opacity-10 border border-gray-800 rounded-md backdrop-filter backdrop-blur-lg bg-gray-200 pb-8'>
+                    <div className='w-full -mt-12 flex justify-center'>
+                        <div className='w-[8rem] ring-2 ring-primary rounded-full overflow-hidden'>
+                            <img alt='Pic' src={'https://templates.iqonic.design/muzik/html/images/login/user.png'} />
```
Comment (Jonath-z) [inline components/Auth/ConfirmEmail/index.jsx:10]: no need of `{}` here . kindly improve it

File: components/Auth/ConfirmEmail/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                        </div>
+                    </div>
+                    <div>
+                        <div className='lg:w-full '>
+                            <img className='' src='https://templates.iqonic.design/muzik/html/images/login/mail.png' alt='email' />
```
Comment (Jonath-z) [inline components/Auth/ConfirmEmail/index.jsx:15]: same thing here

File: components/Auth/ConfirmEmail/index.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <div className="auth h-screen">
+            <div className='flex justify-center items-center w-full h-full'>
+                <div className='lg:max-w-[40%] px-5 w-full bg-opacity-10 border border-gray-800 rounded-md backdrop-filter backdrop-blur-lg bg-gray-200 pb-8'>
+                    <div className='w-full -mt-12 flex justify-center'>
+                        <div className='w-[8rem] ring-2 ring-primary rounded-full overflow-hidden'>
+                            <img alt='Pic' src={'https://templates.iqonic.design/muzik/html/images/login/user.png'} />
+                        </div>
+                    </div>
+                    <div>
+                        <div className='lg:w-full '>
```
Comment (Jonath-z) [inline components/Auth/ConfirmEmail/index.jsx:14]: kindly remove white space

**Review by Jonath-z — COMMENTED** [review]
KIndly Apply these changes

File: pages/confirm-email.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ConfirmEmail from "../components/Auth/confirmEmail";
+
+const ConfirmEmailPage = () => {
+    return (
+        <>
```
Comment (Muhindo-Galien) [inline pages/confirm-email.js:6]: no need of fragment since it's a simple fragment,
kindle remove it

**Review by Muhindo-Galien — CHANGES_REQUESTED** [review]
**Kindly, check the comment added in your code as well as 
check the responsiveness of your page.
upload its picture on both desktop and mobile view for a great review**

**Comment by Derick1530** [comment]
Screenshot 2022-06-27 154859
Screenshot 2022-06-27 160320

**Comment by Muhindo-Galien** [comment]
LGTM

**Pull request merged**